integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -69,6 +69,17 @@ use nearcore::config::{GenesisExt, TESTING_INIT_BALANCE, TESTING_INIT_STAKE};
 use nearcore::{TrackedConfig, NEAR_BASE};
 use rand::Rng;
 
+pub struct PartId {
```

Comment:
here too

Reply:
I was getting error in importing PartId . I am new to Rust ,.I am learning it. I thought that since this project is monorepo we can't import PartId from another package in Rust . Sorry I made such changes . I am really really sorry . I will remove all these duplicated PartId .

Reply:
Sorry I didn't mean to appear too criticizing. It's understandable if you are new to rust. It might be helpful for you to read some tutorials about crates in rust if you are not very familiar with that.